subprojects/core/src/integTest/groovy/org/gradle/internal/operations/notify/BuildOperationNotificationIntegrationTest.groovy

```diff
@@ -135,26 +135,21 @@ class BuildOperationNotificationIntegrationTest extends AbstractIntegrationSpec
     }
 
     // This test simulates what the build scan plugin does.
-    def "can ignore buildSrc events by deferring registration"() {
+    def "listener receives recorded operations happen before registration"() {
```

Comment:
@big-guy ran the performance tests succesfully (see https://builds.gradle.org/viewLog.html?buildId=7781295&tab=buildResultsDiv&buildTypeId=Gradle_Check_PerformanceTestCoordinator )